<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QAlderFramePlayerWidget</class>
 <widget class="QWidget" name="QAlderFramePlayerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>513</width>
    <height>69</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame player</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QToolButton" name="firstFrameButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>Go to start</string>
     </property>
     <property name="text">
      <string>&lt;&lt;</string>
     </property>
     <property name="icon">
      <iconset resource="resources/QAlderWidgets.qrc">
       <normaloff>:/icons/frameToFirst.png</normaloff>:/icons/frameToFirst.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="previousFrameButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>Step backward</string>
     </property>
     <property name="text">
      <string>&lt;</string>
     </property>
     <property name="icon">
      <iconset resource="resources/QAlderWidgets.qrc">
       <normaloff>:/icons/frameBack.png</normaloff>:/icons/frameBack.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="autoRepeat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="playReverseButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>Play backward / Pause</string>
     </property>
     <property name="text">
      <string>PlayBack</string>
     </property>
     <property name="icon">
      <iconset resource="resources/QAlderWidgets.qrc">
       <normaloff>:/icons/framePlayReverse.png</normaloff>
       <normalon>:/icons/pause.png</normalon>:/icons/framePlayReverse.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="playButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>Play forward / Pause</string>
     </property>
     <property name="text">
      <string>Play</string>
     </property>
     <property name="icon">
      <iconset resource="resources/QAlderWidgets.qrc">
       <normaloff>:/icons/framePlay.png</normaloff>
       <normalon>:/icons/pause.png</normalon>:/icons/framePlay.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="nextFrameButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>Step forward</string>
     </property>
     <property name="text">
      <string>&gt;</string>
     </property>
     <property name="icon">
      <iconset resource="resources/QAlderWidgets.qrc">
       <normaloff>:/icons/frameForward.png</normaloff>:/icons/frameForward.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="autoRepeat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="lastFrameButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>Go to end</string>
     </property>
     <property name="text">
      <string>&gt;&gt;</string>
     </property>
     <property name="icon">
      <iconset resource="resources/QAlderWidgets.qrc">
       <normaloff>:/icons/frameToLast.png</normaloff>:/icons/frameToLast.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="repeatButton">
     <property name="toolTip">
      <string>Repeat</string>
     </property>
     <property name="text">
      <string>Repeat</string>
     </property>
     <property name="icon">
      <iconset resource="resources/QAlderWidgets.qrc">
       <normaloff>:/icons/repeat.png</normaloff>:/icons/repeat.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDoubleSpinBox" name="speedFactorSpinBox">
     <property name="toolTip">
      <string>Frame rate</string>
     </property>
     <property name="prefix">
      <string>fps </string>
     </property>
     <property name="decimals">
      <number>0</number>
     </property>
     <property name="minimum">
      <double>1.000000000000000</double>
     </property>
     <property name="maximum">
      <double>60.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>1.000000000000000</double>
     </property>
     <property name="value">
      <double>25.000000000000000</double>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QAlderSliderWidget" name="frameSlider" native="true">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="decimals" stdset="0">
      <number>2</number>
     </property>
     <property name="minimum" stdset="0">
      <double>-1.000000000000000</double>
     </property>
     <property name="maximum" stdset="0">
      <double>-1.000000000000000</double>
     </property>
     <property name="value" stdset="0">
      <double>-1.000000000000000</double>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QAlderSliderWidget</class>
   <extends>QWidget</extends>
   <header>QAlderSliderWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources/QAlderWidgets.qrc"/>
 </resources>
 <connections/>
</ui>
